### ISSUE TITLE
unlockapk.com

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#1817]
### Basic Information

**Test link:** `https://unlockapk.com/dl2/`
**Category:** anti-adblock
**Reported from:** Singapore
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Android 7.1.2; Mobile; rv:68.0) Gecko/68.0 Firefox/68.0
**Extension:** Nano Defender 15.0.0.167

**Comment by llacb47** [comment]
`dropapk.to##+js(aopr, detect)`

**Comment by jspenguin2017** [comment]
@okiehsch

**Comment by okiehsch** [comment]
Does it not work on your end with an up to date filter list?

**Comment by llacb47** [comment]
Tried again and seems to work.

**Comment by jspenguin2017** [comment]
Yea, looks like the website is functioning now.

**Comment by jspenguin2017** [comment]
Yea, looks like it's working now after a delay.

**Comment by okiehsch** [comment]
Yes, I see what you mean with "delay" but the anti adblock message shows up for a short time even if I disable `uBO`.
You mentioned that Nano will fix such annoyances.
`unlockapk.com###detect`
works on my end.
